modules/upnp/doc_classes/UPNP.xml

```diff
@@ -5,6 +5,20 @@
 	</brief_description>
 	<description>
 		Provides UPNP functionality to discover [UPNPDevice]s on the local network and execute commands on them, like managing port mappings (port forwarding) and querying the local and remote network IP address. Note that methods on this class are synchronous and block the calling thread.
+		To forward a specific port:
+		[codeblock]
+		const port=7777 # Specify the port you wish to forward
+		var upnp=UPNP.new() # Create a UPNP class
+		upnp.discover(2000,2,"InternetGatewayDevice") 
+		upnp.add_port_mapping(port) # This line does the forwarding 
+		[/codeblock]
+		
```

Comment:
There shouldn't be empty lines in the XML, they will be parsed as separate paragraphs.